src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String border = "____________________________________________________________\n";
+        System.out.print(border + " Hello! I'm Duke\n" + " What can I do for you?\n" + border);
+        String command;
+        int size = 0;
+        String temp;
+        String tag = "[ ]";
+        String[] temp2 = new String[2];
+        String[] list = new String[100];
```

Comment:
Perhaps declare a final variable for the maximum number of tasks, as follows:
`static final int MAX_LIST_COUNT = 100` ?